### ISSUE TITLE
Towny Costs

### ISSUE THREAD

**Issue by MrSnipeHard** [EarthMC/Issue-Tracker#107]
I will bring up three separate points that are up for independent discussion and modification.

After a brief week or so, we have already seen 12* nations and countless towns sprout up across the map. This has been a great starting point for the establishment of the new map. However, moving forward, the current rates are so low that towns will litter the map, and players are not incentived to remain in a town as a member. Why stay in someone else’s town when one could easily vote for 3 days, mine 2 gold ingots, and found one’s own town? The current rate of 32g should be bumped up to no less than 64g, in my opinion.

To bring back up a controversial but necessary burden, town upkeep should be in place. There should be at least a 1g upkeep for all towns regardless of size. It would encourage individuals to be active at least once every 10 days in order to vote and deposit the free gold. Individuals who are inactive could also go ham mining one day and be inactive for a whole month even with town upkeep. Town upkeep requires people to be dedicated and prepared. Why are these bad traits? Shouldn’t we expect these from our town leaders, and, maybe if they cannot keep up with a small 1g upkeep, they shouldn’t be a town leader.

On that same note, nations have unexpectedly quickly popped up after less than one week of full release. Perhaps, this indicates that the nation founding cost of 512g is too low and should be bumped up to a harder to obtain sum of 1000g. After all, nation construction should be a community operation not a solo act.

All of these points and possible changes comes at odds against the notion of fairness. A solution to this issue would be presenting the current costs as a “Release Special” in order to gather a reasonable presence of towns and nations on the map. An announcement by the server administration could be posted one week ahead of the changes to inform players that these special costs are coming to an end and that the Towny prices will be bumped up to the normal rate.

**Comment by FixTheAdmin** [comment]
I agree, thank you for the detailed post

**Comment by BowShot118** [comment]
All this does is slow down people who have found it hard to find gold. I think that how rare gold is is a deterant on its' own. I know that me and my nation have worked hard since the first day of Beta and don't have anywhere near enough gold for a nation. This makes nations in more remote areas really fucking hard and may make the server stupidly grinding based and no longer enjoyable for many members of the playerbase. I can see the point on increasing town costs but I think pick between increased town costs, I'd say 64g max or adding a small upkeep based in Nuggets. I'd much prefer having a nation upkeep to changing any prices as that means the community has to make sure their nation stays around and not fall inactive after the grind. I'd say 3g/day max for that. Outside of that I believe that it is a horrific idea.

**Comment by sharkfynn1012** [comment]
64 gold for a town is plenty. The nation one, sheesh that is a lot of gold, you’d have to be a large community to get something like that going. Good point on the towns though, and upkeep is a bit iffy, as some people may actually have to go inactive for 10 days or more suddenly or not suddenly. If people want less towns how about set a certain amount of time during the 40 day innacticity period that people have to be in for so that there is less of the rejoin and leave crap with players who are inncactive and just want to keep their towns standing.

**Comment by BowShot118** [comment]
> 64 gold for a town is plenty. The nation one, sheesh that is a lot of gold, you’d have to be a large community to get something like that going. Good point on the towns though, and upkeep is a bit iffy, as some people may actually have to go inactive for 10 days or more suddenly or not suddenly. If people want less towns how about set a certain amount of time during the 40 day innacticity period that people have to be in for so that there is less of the rejoin and leave crap with players who are inncactive and just want to keep their towns standing.

I agree, Classic had its' own ways to controlling towns and how many there are and this map is much bigger so it really shouldn't be an issue rn.

**Comment by ProvingUnique** [comment]
Honestly from a perspective that isn’t about spending as little as possible, my town hustled yesterday to raise plenty of gold to fund the nation. I would say we generated 300 in 3 hours from mining.

I don’t see why the prices shouldn’t be inflated.

Towns-128
Nations-2k
Plot expansion 32

It’s not unrealistic. It can be done. So why not?

**Comment by BowShot118** [comment]
I think that raising town prices to keep people in a town is a pathetic way of trying to keep people in your town when you spam invites too every new player. If you want a large town, work hard and build people's trust. Don't increase the prices for such a petty thing. It just gives older players on tn an advantage over the new ones as they had to pay about 4x Less gold then their predecessors. And claim trading would become very costly if claim pricing was increased.

**Comment by ProvingUnique** [comment]
@BowShot118 I don't think raising prices of a means to keep players locked down in places. 

and its very doable to get that much gold with as little as 4 players

**Comment by BowShot118** [comment]
Someone tell me that Proving. I've mined way to much for the amount I've gained.

**Comment by FutureLemon** [comment]
2K for a nation is just simply overpriced for a nation sure a couple of mods can get it easely, but you are moderators and not regular players that only play 1hour a day. Many people just dont have the time or people to accomplish stuff like this. I think nation's shouldn't be above 1024gold and town's shouldn't be above 64gold and claims are good.

**Comment by MrSnipeHard** [comment]
I agree with what FutureLemon has stated. 64g for founding towns, 1024g for founding nations, and no increase in the current 16g to expand towns. These sound like reasonable figures to have as the normal rate.

I would, however, like to have players bring their thoughts on the table about a town upkeep.

**Comment by BowShot118** [comment]
I'd accept a max of 64g for a town is no upkeep was added. Nations don't hold buildings for value, people's work whereas towns do. Sticking with the inactivity kicker would be the best option imo. Also, if Nation upkeep was removed I'd put up with a nation price of around 700g.

**Comment by MrSnipeHard** [comment]
Bowshot118, I wish you would briefly look over what you say before you post it. It is difficult to pull the meaning out of what you say on all your posts.

It doesn’t have to be an either or. The server can implement both an inactivity kicker and a town/nation upkeep.

**Comment by BowShot118** [comment]
If you read my statement, I would hate the idea of town upkeep due to towns having sentimental value to players. This is why I would prefer an increase in cost over an upkeep. This way you get to keep the town once you founded it. After reading over my statement, it would make sense to anyone who read through it properly instead of assuming it is a passage of hate speach. I am merely sharing my opinion on upkeeps of all kind. 

The second half of my statement was how I'd be happy if nation upkeep was removed in return for a higher price. My recommended price was no more then 700g. Hope I simplified it for you :)

**Comment by Synargle** [comment]
I disagree with increasing town and nation costs and believe things are fine as they are right now. I can sympathise with the prospect of wanting people to work together to make a nation but you will always have those stubborn few who will try to go against what you intend to happen regardless of changes implemented.

I've spoken to people whom think town upkeep is a bad idea and I'm inclined to agree so we should all compromise by at least having a inactivity purger. We had one on classic which would kick players from towns if they haven't been on for 42 days. I'm not certain if that's true for Terra Nova.

**Comment by BowShot118** [comment]
Despite an increase in cost being back imo, it is preferable to a daily cost as it will eventually end up costing more.

**Comment by Taicamila** [comment]
> 
> 
> Despite an increase in cost being back imo, it is preferable to a daily cost as it will eventually end up costing more.

I hope it doesn't, tbh. I'm struggling enough as it is and we were just robbed of our gold recently which put us back about a month because of our inability to find much of it.

**Comment by BowShot118** [comment]
If possible, could you dm me on discord - same name. But, I see your worries and we are alsp struggling to get gold but I'd rather not have too keep worrying about it and possibly having to make a new nations once again.

**Comment by MineHero43** [comment]
@MrSnipeHard if I may ask, do you own a town or nation on Terra Nova and what is your IGN?

**Comment by BowShot118** [comment]
His IGN is the same as his GitHub name

**Comment by MineHero43** [comment]
Nation? Town?

**Comment by BowShot118** [comment]
Idk name. Do a /res ingame

**Comment by MineHero43** [comment]
Considering he owns a small one chunk town part of the nation lead by the elitists who ran the economy on classic, I wonder if he has actually been through the struggle of mining and trying to find gold.

I suggest that he try and find gold to form a nation and endure the frustration which many players currently are and then have to deal with people who have probably been mining once or twice, wishing for their prices to be increased

**Comment by BowShot118** [comment]
Loving the subtle mention there MineHero, but I do agree. If you haven't wasted your life away mining on Terra Nova, go do that then come back and tell us this is all a good idea.

**Comment by ProvingUnique** [comment]
sly your banned. 

if prices were higher, players would invest more time into it. like all of us already are

**Comment by KarlOfDuty** [comment]
I think towns are good just the way they are. Players should have to make a choice between being safe by joining a town or dedicating themselves to making their own. At this point I think this is well balanced especially as pvp will be on at some point and keepinventory turned off. It took me a few days to make my town on my own and that is quite some time to be unprotected. If you are a few friends who vote to immediately get enough gold then you deserve to quickly have a town as towns are supposed to be for small communities playing together.

However, a nation was created literally within the first two days which is not at all what we had in mind on the new server. We wanted nations to be huge investments made by a large number of players. It should have taken way longer for a nation to be made, especially one that small.

**Comment by KarlOfDuty** [comment]
Whoops

**Comment by MineHero43** [comment]
@ProvingUniqie

He’s banned but still more relevant than you are.

Also it’s actually quite the contrary, higher prices means players, especially new players will invest less time as they’re much faster to get fed up of not finding any gold.

The challenge for a nation to hard enough already, you would know that if you actually paid attention to the community and their complaints for once. 

Players are frustrated, why would we want to go as far as to harming the playerbase by increasing their prices to an almost unplayable, unenjoyable extent?

**Comment by KarlOfDuty** [comment]
Stop being a dick about it or it is you who will be removed from the conversation.

Nations are not supposed to be for new players, nations are supposed to be an end-game goal that only a large number of players who dedicate their gold towards it can accomplish. The fact that you were able to make a nation in less than 24 hours of mining means that the nations are completely un-balanced and must be made much harder to create.

**Comment by MineHero43** [comment]
I’m not being a dick at all, is what I said even close to being a dick?

Defending players who are only able to gather 7 gold per day if lucky makes me a dick? Trying to increase their prices and suffering actually seems to match the very word you just spewed at me.

I may be able to spend my entire weekend on the server but others clearly aren’t.

I suggest that you try forming a nation with the free time that you have before agreeing to increasing the prices to such high degrees.

**Comment by KarlOfDuty** [comment]
I was obviously talking about your comments directed to ProvingUnique.

**Comment by MineHero43** [comment]
I wouldn’t refer to them as the word which you used either, in fact you making that comment actually makes you seem more like one.

I suggested that he pay attention to the vast number of complaints made by the community, is that too much to ask from a staff member?

**Comment by KarlOfDuty** [comment]
In any case, I don't get your reasoning here at all. You yourself made a nation within 24 hours. Assuming you literally spent every second of that time mining that would mean that a single person spending 3 hours mining a day gets a nation after a week. If you just have 3 people in your town you just need to spend a single hour a day mining. That is way less than it should take. It should be 10+ players coming together to dedicate themselves to making a nation over some time, not something one person could do in a day off work/school.

**Comment by MineHero43** [comment]
What I did was actually quite unhealthy as it involved staring at a screen from 3PM to a late 2AM, non stop. That for 2 days. I hope you don’t encourage and expect that from everyone else who are struggle right now.

I also hope you’re able to see that players simply aren’t interested in doing what I did. Yes, nations should be a challenge. Quite frankly I easily found over a thousand gold that weekend, were the prices 1k gold I would’ve still formed Britain in under a days worth of playtime, maybe a little more by one or two hours.

**Comment by MineHero43** [comment]
Are you suggesting that I somehow cheated to get my wealth? That’s actually despicable of you.

Go and have your witch hunt, the only thing you’ll find is me joking about Sly giving me the power to dupe. Then again, knowing you that’s probably sufficient evidence to permanently remove my name from the face of the server.

**Comment by KarlOfDuty** [comment]
Excuse me? I was suggesting that other players will have similar results to you because all gold spawns randomly and evenly.

**Comment by MineHero43** [comment]
I’ll take the L, however your comment could easily be interpreted as what I believed it to be.

**Comment by BowShot118** [comment]
I think MineHero was just lucky. Despite the claims of it being evenly spread. It clearly isn't. MineHero is having a blast mining for a decent amount of gold whereas other players are struggling. But, MineHero did comment is walking past gold ore in walls in mineshafts which is unusual. I'm just suggesting that maybe due to the intenseag at the weekends, the anti X-Ray plugin may have bugged out which could explain such large amounts of gold in commonly mined areas and such low amounts in fairly untouched areas. Just a suggestion. In reference to Proving, that's low, and, I'm already sick and tired of grinding here due to the amount I have done in the past few days. Please do bare in mind that not every player has Enchanted  tools in which they can increase their mining speed. This may be one of the contributing factors to MineHero's early success.

**Comment by KarlOfDuty** [comment]
It would seem like it has more to do with caves rather than the gold spawns. Caves seem more common in some areas of the world than others. 

In any case, as I've said before, nations are not supposed to be something players can get quickly and not by themselves. Different towns of lots of players should come together to build a nation, not one person.

**Comment by SpookyScoops** [comment]
There's no reasonable argument to increase town prices, because the only alternative to founding a town is joining a large town and sitting in a one chunk plot till you get enough vote gold to venture out. 32 gold is reasonable because it is an achievable amount to grind out in the wilderness. 

The real question is how much stars we would like to see on the dynmap. Should we have 50 stars in Europe on the map or only a dozen or so? I believe that, since this is just minecraft after all, that people are going to grind out whatever amount of gold it takes because many people don't want to have to be pressured into having to be part of a large nation. People want freedom of sorts and to be able to choose their own fate on the server. It's pretty unrealistic to expect everyone to just settle and join a massive nation just because they have the manpower to achieve a nation.

This is coming from a guy that grinded for a week just to get his little piece of freedom.

**Comment by TypicalFernie** [comment]
brb going to Ottawa

**Comment by Toadally** [comment]
Here in Iceland, it's almost completely dry. Mining trips by a few of hours have totaled well over 20 hours and we're still just barely half way to starting a nation.

**Comment by sharkfynn1012** [comment]
True that, I was unaware of Ottawa’s gold. At the time. One can only pay attention to so many messages.

**Comment by ProvingUnique** [comment]
I just view nations as long term goals

**Comment by MSatrokrm** [comment]
> I just view nations as long term goals
> […](#)
> On Nov 10, 2018, at 8:25 PM, Kauê The Trekker ***@***.***> wrote: Honestly from a perspective that isn’t about spending as little as possible, my town hustled yesterday to raise plenty of gold to fund the nation. I would say we generated 300 in 3 hours from mining. I don’t see why the prices shouldn’t be inflated. Towns-128 Nations-2k Plot expansion 32 It’s not unrealistic. It can be done. So why not? what the heck? we have been mining for 2 days and I'm sure we haven't reached even a stack of gold. — You are receiving this because you were mentioned. Reply to this email directly, view it on GitHub, or mute the thread.

Yes, forming a nation is a long-term goal. However, the prices you've stated are just bloody ludicrous.

You cannot expect other players to have the same luck as you had with finding gold, nor can you expect them to play as much as you do.

My personal opinion: I think that the towny prices are fine where they currently are.
